Skyra/presence.ts

```diff
@@ -0,0 +1,46 @@
+var presence = new Presence({
+  clientId: "266624760782258186",
+});
```

Comment:
remove comma

Reply:
Yeah sure. Just as an aside though, the commas were added by Prettier. If you do not want trailing commas you may want to change `es5` to `none` in the `.prettierrc.json` file, right now it's `es5` which adds trailing commas where it's valid to have them as per the ES5 spec.

For more info see Prettier options: https://prettier.io/docs/en/options.html#trailing-commas